src/index.scss

```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0;
+  padding: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
```

Comment:
```suggestion
```

I know this was added in another commit, but since we're editing this file then can we remove this since it's already applied by Bootstrap?

Reply:
Thank you for taking your time and giving us these very helpful comments. I really appropriate I have learned a lot of stuff from your comments and feedback. :)

Reply:
Glad the comments have been very helpful!